apps/docs/blog/2022-09-10-webops.md

```diff
@@ -0,0 +1,27 @@
+---
+description: WebOps can now reuse pull requests' cache for new commit deploys.
+tags: [webops]
+---
+
+# New environment flag improves deploy cache use
+
+WebOps can now reuse pull requests' cache for new commit deploys. This is because of a new environment flag you can set in the `vtex.env` file of your FastStore project: `USE_BUILD_CACHE`.
+
+## What has changed?
+
+The new `USE_BUILD_CACHE` flag replaces the previously used `USE_FRAMEWORK_CACHE` and `USE_NODE_MODULES_CACHE` flags.
+
+To exemplify the improvement, consider the git example below.
+
+```
+master: commit-sha-1
+PR 233:
+       - commit-sha-2
+       - commit-sha-3
+```
+
+Previously, the build correponding to `commit-sha-3` would use `commit-sha-1` cache. Now it uses the cache generated for `commit-sha-2`, unless it does not exist, in which case it uses `commit-sha-1` cache.
```

Comment:
```suggestion
Previously, the build correponding to `commit-sha-3` would use `commit-sha-1` cache. Now it uses the cache generated for `commit-sha-2`. In case the `commit-sha-2` does not exist, the build would use the `commit-sha-1` cache.
```